library/modules/Units.cpp

```diff
@@ -1932,19 +1939,6 @@ bool Units::isGoalAchieved(df::unit *unit, size_t goalIndex)
         && unit->status.current_soul->personality.dreams[goalIndex]->flags.whole != 0;
 }
 
-std::string Units::getSquadName(df::unit *unit)
```

Comment:
how come removing this code resulted in an *increase* of included header files in Units.cpp?